contentcuration/contentcuration/assets/shared/views/AppBar.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <VToolbar app dark color="purple" :tabs="Boolean($slots.tabs)">
+    <VToolbarSideIcon href="/channels">
+      <VImg maxHeight="35" contain :src="require('../images/kolibri-logo.svg')" />
+    </VToolbarSideIcon>
+
+    <VToolbarTitle class="white--text">
+      Kolibri Studio Beta
```

Comment:
I removed the translation wrapping on beta, as we should either translate the whole thing or none of it.